elm.json

```diff
@@ -12,6 +12,8 @@
         "Nri.Ui.AssignmentIcon.V1",
         "Nri.Ui.BannerAlert.V6",
         "Nri.Ui.Button.V5",
+        "Nri.Ui.Button.V6",
```

Comment:
Okay so because of https://github.com/NoRedInk/noredink-ui/pull/493 I think we can now safely remove Button.V6, and plan to update the one spot that needs a special link type.

Reply:
sounds like a plan